great_expectations/execution_engine/sparkdf_execution_engine.py

```diff
@@ -260,6 +260,7 @@ def get_batch_data_and_markers(
             reader_options: dict = batch_spec.reader_options or {}
             path: str = batch_spec.path
             azure_url = AzureUrl(path)
+            # TODO <WILL> 202209 - Add `schema` definition to Azure like PathBatchSpec below
```

Comment:
If you add a TODO to the code, I think adding a ticket and referencing it is a good strategy or this outstanding work may get lost.

Reply:
Thanks for the suggestion. Now being tracked by GREAT-1224

Reply:
Can you add that ticket number to the comment?

Reply:
Added the ticket number to the comment!